website_react/src/components/inputs/InputENUM.jsx

```diff
@@ -0,0 +1,29 @@
+const InputENUM = (props) => {
+    return (
+        <div className="form-input-div" key={props.index}>
+            <label className="input-label">{props.formFields.title}</label>
+            {props.formFields.options.map((value, index) => {
```

Comment:
1 rule for react components (my rule)
No logic. No loops. No ifs. inside the JSX.  
suggestions 
```
const renderOption = (option) = > ...return jsx here 

const options = props.options.map(renderOption)

    return (
         <div className="form-input-div" key={props.index}>
             <label className="input-label">{props.formFields.title}</label>
             {options}.
           </div>)
```